### ISSUE TITLE
Unhandled message

### ISSUE THREAD

**Issue by Benjamin-Etheredge** [hsahovic/poke-env#200]
Was getting this error periodically while training. I patched `abstract_battle.py` to work around it for now, but I don't know what to do with it yet.

```
NotImplementedError: ['', '-swapsideconditions']
2021-09-17 07:01:13,651 - RandomPlayer 1 - ERROR - Unhandled exception raised while handling message:
>battle-gen8randombattle-2713
|
|t:|1631862073
|move|p2a: Cinderace|Court Change||[still]
|-start|p2a: Cinderace|typechange|Normal|[from] ability: Libero
|-swapsideconditions
|-fail|p2a: Cinderace
|move|p1a: Corsola|Recover|p1a: Corsola
|-heal|p1a: Corsola|100/100 psn
|
|-damage|p1a: Corsola|88/100 psn|[from] psn
|upkeep
|turn|32
```

**Comment by hsahovic** [comment]
Hey @Benjamin-Etheredge,

That's a good catch. This happened with standard random battles with showdown's latest version?

**Comment by pmariglia** [comment]
@hsahovic new way to handle courtchange

https://github.com/smogon/pokemon-showdown/pull/8287

**Comment by hsahovic** [comment]
Ah, that makes sense. I'll go over it over the weekend.

**Comment by Benjamin-Etheredge** [comment]
I was *just* about to make another issue due to the following error, but it turns out it's likely related to this one. The move used looks to be *swap court*.

Got an unknown condition. I
```
KeyError: <SideCondition.AURORA_VEIL: 2>
2021-09-17 14:42:27,984 - SimpleRLPlayer 1 - ERROR - Unhandled exception raised while handling message:
>battle-gen8randombattle-3921
|
|t:|1631889747
|switch|p2a: Abomasnow|Abomasnow, L82, F|100/100
|move|p1a: Cinderace|Court Change|p1a: Cinderace
|-swapsideconditions
|-activate|p1a: Cinderace|move: Court Change
|
|-weather|Hail|[upkeep]
|-damage|p1a: Cinderace|15/240|[from] Hail
|-sideend|p1: SimpleRLPlayer 1|move: Aurora Veil
|upkeep
|turn|15
```

**Comment by hsahovic** [comment]
This should be fixed by 4ea2ffdea4f3df8e6f3c7a0264cd6e068ad80e9c - It will be part of the next release.

**Comment by hsahovic** [comment]
This is fixed in release `0.4.19`.
